src/js/components/Menu/Menu.js

```diff
@@ -210,9 +220,12 @@ const Menu = forwardRef((props, ref) => {
         active={activeItemIndex === controlButtonIndex}
         focusIndicator={false}
         hoverIndicator="background"
+        label={theme.button.default && !children ? menuLabel : undefined}
+        icon={theme.button.default && !children ? menuIcon : undefined}
         plain={plain}
         onClick={onDropClose}
         onFocus={() => setActiveItemIndex(controlButtonIndex)}
+        reverse={theme.button.default && !children ? true : undefined}
```

Comment:
```suggestion
        reverse={theme.button.default && !children}
```
try this

Reply:
Pull out these 3 variables and any other shared props and spread across controlMirror and DropButton